utils/python/update_acme_tests.py

```diff
@@ -48,8 +48,8 @@
                              "ERR.f45_g37_rx1.A",
                              "ERP.f45_g37_rx1.A",
                              "SMS_D_Ln9.f19_g16_rx1.A",
-                             "DAE.f19_f19.A",
-                             "PRE.f19_f19.ADESP")
```

Comment:
If you have any run directories lying around, I would love to see a grep 'atm: model date' from them to see how far they made it.